extensions/poetry/PhylumExt.toml

```diff
@@ -4,5 +4,5 @@ entry_point = "main.ts"
 
 [permissions]
 run = ["poetry"]
-read = ["./poetry.lock", "./pyproject.toml"]
+read = true,
```

Comment:
Why is this still modified? Also I'm not sure the comma is even valid toml? I'd assume this was a bug during rebase?

Reply:
Reverted